components/core/.clang-format

```
@@ -75,7 +75,7 @@ IncludeCategories:
   # Library headers. Update when adding new libraries.
   # NOTE: clang-format retains leading white-space on a line in violation of the YAML spec.
   - Regex: "<(absl|antlr4|archive|boost|bsoncxx|catch2|curl|date|fmt|json|log_surgeon|mariadb\
```

Do we need to remove `boost` here since it got replaced?
```suggestion
  - Regex: "<(absl|antlr4|archive|bsoncxx|catch2|curl|date|fmt|json|log_surgeon|mariadb\
```

---

We have other boost libraries such as `boost::filesystem`